miscelanea/004-route-traffic/poc-004-4c.yaml

```diff
@@ -0,0 +1,190 @@
+        - name: "ECHO_PORT"
+          value: "636"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 636
+          protocol: TCP
+    - name: kerberos-tcp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "88"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 88
+          protocol: TCP
+    - name: kerberos-admin-tcp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "464"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 464
+          protocol: TCP
+    - name: kerberos-udp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "88"
+        - name: "ECHO_PROTOCOL"
+          value: "udp"
+      ports:
+        - containerPort: 88
+          protocol: UDP
+    - name: kerberos-admin-udp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "464"
+        - name: "ECHO_PROTOCOL"
+          value: "udp"
+      ports:
+        - containerPort: 464
+          protocol: UDP
+---
+# https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+# https://kubernetes.io/docs/concepts/services-networking/service/#externalname
+# https://kubernetes.io/docs/concepts/services-networking/service/#external-ips
+# https://docs.openshift.com/container-platform/4.6/networking/configuring_ingress_cluster_traffic/configuring-externalip.html#nw-externalip-about_configuring-externalip
+# https://docs.openshift.com/container-platform/4.6/installing/installing_openstack/installing-openstack-installer-custom.html#installing-openstack-installer-custom
+# https://docs.openshift.com/container-platform/3.4/dev_guide/expose_service/expose_internal_ip_service.html
+# Egress is for traffic from cluster to outside
+# https://www.openshift.com/blog/how-to-enable-static-egress-ip-in-ocp
+# https://docs.openshift.com/container-platform/4.4/networking/configuring_ingress_cluster_traffic/configuring-externalip.html
+# Allocated floating IP 10.0.133.188
+# The initial floating IP for apps: 10.0.135.189
+apiVersion: v1
+kind: Service
+metadata:
+  name: poc-004-4c
+  labels:
+    app: poc-004-4c
+spec:
+  type: "NodePort"
+  # externalTrafficPolicy: Cluster
+  selector:
+    app: poc-004-4c
+    role: services
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      nodePort: 31080
```

Comment:
https://kubernetes.io/docs/concepts/services-networking/service/#nodeport

If I try to use a port out of the range 30000-32767, the service object is rejected.